chapter02/sxg/src/main/java/com/github/sebin/chapter02/User.java

```diff
@@ -0,0 +1,36 @@
+package com.github.sebin.chapter02;
+
+public class User {
+	private int customerID;
+	private String username;
+	private String password;
+	private String fullname;
```

Comment:
When the customer changes his name, create a new User object. Let the old one get garbage collected.

I want to have classes as much as possible to be immutable.